### PR TITLE
Update about us

### PR DESCRIPTION
**Pull request by ebarry**
Hi all, this PR does not have the boxes checked, can it still be reviewed? 
It adds one link to the About Us dropdown in coordination with the nonprofit. 

Fixes #0000 (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updation
* [ ] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Review by daemon1024 — APPROVED** [review]
image

Seems to be working great 🚀

**Comment by ebarry** [comment]
great thanks!
Do you know when this will go to stable, and then to production?
--

+1 336-269-1539 / @lizbarry <http://twitter.com/lizbarry> / lizbarry.net

**Comment by daemon1024** [comment]
It should be on stable as soon as someone with authorization pushes the `Merge Button`. For the production, I am not sure about it but it's usually within a week or two after it's pushed to stable.

**Review by jywarren — APPROVED** [review]

**Pull request merged**
